pylint/lint/pylinter.py

```diff
@@ -617,8 +619,9 @@ def set_option(self, optname, value, action=None, optdict=None):
                 except KeyError:
                     meth = self._bw_options_methods[optname]
                     warnings.warn(
-                        "%s is deprecated, replace it by %s"
-                        % (optname, optname.split("-")[0]),
+                        "{} is deprecated, replace it by {}".format(
+                            optname, optname.split("-")[0]
+                        ),
```

Comment:
As this isn't related, it would be best to exclude the change here. Even if technically it's an improvement.

Reply:
Want me to create another commit/PR to do this? Or just let it be?

Reply:
If you really like to, you can create another one. But if you ask me, I would let it be. It doesn't harm anybody and is still valid code.